pkg/pillar/evetpm/tpm.go

```diff
@@ -76,6 +77,28 @@ const (
 	// TpmSavedDiskSealingPcrs is the file that holds a copy of PCR values
 	// at the time of generating and sealing the disk key into the TPM.
 	TpmSavedDiskSealingPcrs = types.PersistStatusDir + "/sealingpcrs"
+
+	// TpmEvtLogBinFile is a kernel exposed variable that contains the
+	// TPM event log.
+	TpmEvtLogBinFile = "binary_bios_measurements"
+
+	// TPMEvtLogDir is directory that TPM event log resides in.
+	TPMEvtLogDir = "/hostfs/sys/kernel/security/tpm*"
+
+	// TpmEvtLogSealSuc is files that holds a copy of event log at the time
```

Comment:
Suc is pronounced "sucks", right?
Sorry for being picky about names, but Success or Ok would be better.

Reply:
fixed.